raft/raft.go

```diff
@@ -157,7 +161,15 @@ type raft struct {
 	// New configuration is ignored if there exists unapplied configuration.
 	pendingConf bool
 
-	elapsed          int // number of ticks since the last msg
+	// number of ticks since it reached last electionTimeout
```

Comment:
`reached last electionTimeout` is inaccurate IMO. the var could be reset when receiving any message from leader.